infra/azure/scripts/generate-env.js

```
@@ -290,8 +298,8 @@ console.log('');
 console.log('  2. Configure GitHub secrets:');
 console.log(`     bash ${secretsFile}`);
 console.log('');
-console.log('  3. Set up Azure AD B2C for authentication:');
-console.log('     ./scripts/setup-b2c.sh phoenixrooivalkb2c');
+console.log('  3. Set up Azure Entra ID for authentication:');
```

ESLint

[error] Replace `'··3.·Set·up·Azure·Entra·ID·for·authentication:'` with `"··3.·Set·up·Azure·Entra·ID·for·authentication:"`

(prettier/prettier)